src/pages/settings/Security/TwoFactorAuth/IsEnabledPage.js

```diff
@@ -0,0 +1,69 @@
+import React, {useState} from 'react';
+import {Text, View} from 'react-native';
+import HeaderWithCloseButton from '../../../../components/HeaderWithCloseButton';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import ROUTES from '../../../../ROUTES';
+import Section from '../../../../components/Section';
+import * as Illustrations from '../../../../components/Icon/Illustrations';
+import * as Expensicons from '../../../../components/Icon/Expensicons';
+import themeColors from '../../../../styles/themes/default';
+import styles from '../../../../styles/styles';
+import ConfirmModal from '../../../../components/ConfirmModal';
+
+const defaultProps = {};
+
+function IsEnabledPage(props) {
+    const [isConfirmModalVisible, setIsConfirmModalVisible] = useState(false);
+
+    return (
+        <ScreenWrapper>
+            <HeaderWithCloseButton
+                title={props.translate('twoFactorAuth.headerTitle')}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_SECURITY)}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+            />
+            <Section
+                title={props.translate('twoFactorAuth.twoFactorAuthEnabled')}
+                icon={Illustrations.ShieldYellow}
+                menuItems={[
+                    {
+                        title: props.translate('twoFactorAuth.disableTwoFactorAuth'),
+                        onPress: () => {
+                            setIsConfirmModalVisible(true);
+                        },
+                        icon: Expensicons.Close,
+                        iconFill: themeColors.danger,
+                        wrapperStyle: [styles.cardMenuItem],
+                    },
+                ]}
+                containerStyles={[styles.twoFactorAuthSection]}
+            >
+                <View style={styles.mv3}>
+                    <Text style={styles.textLabel}>{props.translate('twoFactorAuth.whatIsTwoFactorAuth')}</Text>
+                </View>
+            </Section>
+            <ConfirmModal
```

Comment:
Let's pass a `danger` prop and make the text as "Disable", to achieve this

<img width="384" alt="Screenshot 2023-04-27 at 6 29 26 PM" src="https://github.com/Expensify/App/assets/32012005/edf33d78-e248-4015-9d4a-14626d9d9298">

Reply:
Done 👍🏻